executor/point_get.go

```diff
@@ -336,7 +342,27 @@ func (e *PointGetExecutor) get(ctx context.Context, key kv.Key) ([]byte, error)
 		}
 		// fallthrough to snapshot get.
 	}
-	return e.snapshot.Get(ctx, key)
+
+	isLocked := e.tblInfo.IsLocked()
```

Comment:
We need to check the lock type.

Reply:
fix, if query with write lock will query from kv.